### PR TITLE
Remove from/to LocalTimestamp deprecated query params

### PR DESCRIPTION
**Pull request by agusaldasoro**
Remove deprecated old params `fromLocalTimestamp` & `toLocalTimestamp` from `/deployments` & `/pointer-changes`

### CODE REVIEW

**Comment by menduz** [comment]
Please consider updating this first https://github.com/decentraland/snapshots-fetcher/blob/66b24b2b2a8d5df11a6757baed52f7ac2a5e6ed5/src/client.ts#L46

**Comment by agusaldasoro** [comment]
> Please consider updating this first https://github.com/decentraland/snapshots-fetcher/blob/66b24b2b2a8d5df11a6757baed52f7ac2a5e6ed5/src/client.ts#L46

There's no need to. This PR affects only clients using `toLocalTimestamp` or `fromLocalTimestamp` query parameters if the client is already using `from`/`to` + `sortingField`, then the query is "smart" enough to do what it needs: https://github.com/decentraland/catalyst/blob/main/content/src/logic/database-queries/deployments-queries.ts#L175

File: content/src/controller/Controller.ts [hunk 1]
```diff
@@ -341,14 +337,10 @@ export class Controller {
...
 
-    // TODO: remove this when to/from localTimestamp parameter is deprecated to use to/from
-    const fromFilter = from ?? fromLocalTimestamp
-    const toFilter = to ?? toLocalTimestamp
-
     const requestFilters = {
       entityTypes: entityTypes as EntityType[] | undefined,
-      from: fromFilter,
-      to: toFilter,
+      from: from,
```
Comment (jmoguilevsky) [inline content/src/controller/Controller.ts:342]:
```suggestion
      from,
```

File: content/src/controller/Controller.ts [hunk 2]
```diff
@@ -341,14 +337,10 @@ export class Controller {
...
+      to: to,
```
Comment (jmoguilevsky) [inline content/src/controller/Controller.ts:343]:
```suggestion
      to,
```

File: content/src/controller/Controller.ts [hunk 3]
```diff
@@ -485,27 +473,13 @@ export class Controller {
...
-      (!sortingField || sortingField == SortingField.LOCAL_TIMESTAMP) && toLocalTimestamp ? toLocalTimestamp : to
-
     const requestFilters = {
       pointers,
       entityTypes: entityTypes as EntityType[],
       entityIds,
       onlyCurrentlyPointed,
-      from: fromFilter,
-      to: toFilter
+      from: from,
```
Comment (jmoguilevsky) [inline content/src/controller/Controller.ts:481]:
```suggestion
      from,
```

File: content/src/controller/Controller.ts [hunk 4]
```diff
@@ -485,27 +473,13 @@ export class Controller {
...
+      to: to
```
Comment (jmoguilevsky) [inline content/src/controller/Controller.ts:482]:
```suggestion
      to
```

**Review by jmoguilevsky — APPROVED** [review]

**Pull request merged**